vc17/theforgottenserver.vcxproj

```diff
@@ -101,7 +101,7 @@
       <EnableParallelCodeGeneration>true</EnableParallelCodeGeneration>
       <EnableEnhancedInstructionSet>AdvancedVectorExtensions</EnableEnhancedInstructionSet>
       <AdditionalIncludeDirectories>$(VcpkgRoot)include\luajit;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <LanguageStandard>stdcpp20</LanguageStandard>
+      <LanguageStandard>stdcpp23</LanguageStandard>
```

Comment:
for the record it's "std:c++23preview" for visual studio right now, or technically, you can use "std:latest", but it would include some features already implemented for C++26.... we are all still looking super hard at Microsoft about them dragging their feet on this after having been the first one to have it ready for the two previous language versions (C++17/20)